# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/ladybird/hotfix-inline-to_numeric.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- serenity/Userland/Libraries/LibJS/Runtime/Value.cpp.vanilla
+++ serenity/Userland/Libraries/LibJS/Runtime/Value.cpp
@@ -586,7 +586,7 @@
 }
 
 // 7.1.3 ToNumeric ( value ), https://tc39.es/ecma262/#sec-tonumeric
-FLATTEN ThrowCompletionOr<Value> Value::to_numeric_slow_case(VM& vm) const
+ALWAYS_INLINE ThrowCompletionOr<Value> Value::to_numeric_slow_case(VM& vm) const
 {
     // 1. Let primValue be ? ToPrimitive(value, number).
     auto primitive_value = TRY(to_primitive(vm, Value::PreferredType::Number));
